README.md

```diff
@@ -12,7 +12,6 @@ For more details and an introduction into the inner workings of ODL, please refe
 
 Highlights
 ==========
-- Well-tested data containers based on NumPy or CUDA allow high performance computing with minimal effort.
```

Comment:
Weird that we still had this on the front page README

Reply:
Probably should have been in another PR but sure :)